pkg/metrics/metrics.go

```diff
@@ -0,0 +1,112 @@
+package metrics
+
+import (
+	"net/http"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rancher/rancher/pkg/auth/util"
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/types/config"
+	authV1 "k8s.io/api/authorization/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	prometheusMetrics = false
+
+	gcInterval = time.Duration(60 * time.Second)
```

Comment:
any benefit to let someone set this by config or envvar?

Reply:
Not that I can think of? I could be persuaded to do so but not sure it's actually useful.